### PR TITLE
Feature/add about section

### PR DESCRIPTION
**Pull request by vikram-rawat**
### Changes

With this PR we have an about page that appears once you press the info ICON on the navbar

The about section describes the technologies used in the project and created by appsilon.

Please run the app manually to test it.

### CODE REVIEW

File: inst/examples/dashboard/app.R [hunk 1]
```diff
@@ -18,7 +19,11 @@ examplePages <- imap(examples, function(example, name) {
 })
 names(examplePages) <- NULL
 
-pages <- c(list(route("/", homePage)), examplePages)
+pages <- c(list(route("/", homePage)),
+  list(route("about", aboutPage)),
+  examplePages
+)
```
Comment (kamilzyla) [inline inst/examples/dashboard/app.R:25]: Slightly cleaner:
```suggestion
pages <- c(
  list(
    route("/", homePage),
    route("about", aboutPage)
  ),
  examplePages
)
```

File: inst/examples/dashboard/about.R [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        ),
+        Text(
+          " With Shiny Router we can actually create a shiny App That
+          behaves like a multipage App. Where you can direct user from
+          one point to another through using hrefs and hyperlinks.
+          This is something we never had in shiny and you might have
+          found yourself stuck in a project where people want to share
+          URL of only one perticular Page. With Shiny Router we try to
+          overcome these challenges in a shiny App. Try the package
+          and Give us your feedback So that we can grow together..."
```
Comment (kamilzyla) [inline inst/examples/dashboard/about.R:162]: The "single-page app" and "multi-page app" are technical terms with specific meaning. In fact all Shiny apps are single-page apps (only one request is made at the beginning to load the page, which is then dynamically modified) and shiny.router doesn't change that.
```suggestion
          "With shiny.router you can introduce routing to your Shiny application:
          use the URL to direct user from one view to to another.
          This not only simplifies the implementation
          but also allows the users to bookmark or share the URL of a specific view."
```

File: inst/examples/dashboard/about.R [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+          variant = "large", "Check the rest of the App to Learn More...",
+          block = TRUE
+        ),
+        Text(
+          "We believe that a great UI plays a huge role in the success of
+          application projects. shiny.fluent gives your apps: - beautiful,
+          professional look - rich set of components easily usable in Shiny -
+          fast speed of development that Shiny is famous for. As Fluent UI is
+          built in React, shiny.fluent is based on another package called
+          shiny.react, which allows for using React libraries in Shiny."
```
Comment (kamilzyla) [inline inst/examples/dashboard/about.R:77]: A list needs reformatting in this text...
```suggestion
          "We believe that a great UI plays a huge role in the success of
          application projects. shiny.fluent gives your apps: beautiful,
          professional look, rich set of components easily usable in Shiny
          and fast speed of development that Shiny is famous for. As Fluent UI is
          built in React, shiny.fluent is based on another package called
          shiny.react, which allows for using React libraries in Shiny."
```

**Review by kamilzyla — CHANGES_REQUESTED** [review]
Looking good, just a few nitpicks.

File: inst/examples/dashboard/app.R (hunk 1 above)
Comment (vikram-rawat) [reply inst/examples/dashboard/app.R]: Thanks I didn't know I could do this. FIXED!!

File: inst/examples/dashboard/about.R (hunk 2 above)
Comment (vikram-rawat) [reply inst/examples/dashboard/about.R]: THanks for the explanation added the text as it is.

File: inst/examples/dashboard/about.R (hunk 3 above)
Comment (vikram-rawat) [reply inst/examples/dashboard/about.R]: Changed the text

**Review by kamilzyla — APPROVED** [review]

**Pull request merged**